notebooks/241-freevc/241-freevc.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>I think we should not comment this part, we should use this interface for audio, just use our audio as default choice if user does not have own one</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>